generativeai/src/main/java/com/google/ai/client/generativeai/Chat.kt

```diff
@@ -101,43 +124,19 @@ class Chat(private val model: GenerativeModel, val history: MutableList<Content>
     attemptLock()
 
     val flow = model.generateContentStream(*history.toTypedArray(), prompt)
-    val bitmaps = LinkedList<Bitmap>()
-    val blobs = LinkedList<BlobPart>()
-    val text = StringBuilder()
-
+    val tempHistory = LinkedList<Content>()
```

Comment:
In Kotlin, we try not to use inherited classes for lists. You generally wanna use `List` and `MutableList` directly, as it's more straight forward to read (there's no cognitive overhead in thinking about implementation specific details).
```suggestion
    val tempHistory = mutableListOf<Content>()
```